samples/drivers/gpio/src/main.c

```diff
@@ -91,11 +95,15 @@ void main(void)
 	}
 
 	while (1) {
-		printk("Toggling pin %d\n", GPIO_OUT_PIN);
+		printk("Toggling pin %s.%d\n",
+		       gpio_out_dev->config->name,
+		       GPIO_OUT_PIN);
 
 		ret = gpio_pin_write(gpio_out_dev, GPIO_OUT_PIN, toggle);
 		if (ret) {
-			printk("Error set pin %d!\n", GPIO_OUT_PIN);
+			printk("Error set pin %s.%d!\n",
+			       gpio_out_dev->config->name,
+			       GPIO_OUT_PIN);
 		}
 
 		if (toggle) {
```

Comment:
Could you also use toggling with `xor` instead of the `if-else`?

`toggle ^= 1;`

Reply:
Sure, but that would (should) be another, separate PR then.

Reply:
What you've said, or I was thinking of transforming the PR into a cleanup of this sample. But what suits best.

Reply:
If you merge this I promise to create a follow-up PR with the cleaning up :)